spec/views/events/new.html.erb_spec.rb

```diff
@@ -7,7 +7,7 @@
     render
 
     assert_select 'form[action=?][method=?]', events_path, 'post' do
-      assert_select 'input#event_hashtag[name=?]', 'event[hashtag]'
+      assert_select 'input#event_hashtag[name=?]', 'event[hash_tag]'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.